cmd/tendermint/commands/light.go

```diff
@@ -46,185 +94,134 @@ When /abci_query is called, the Merkle key path format is:
 Please verify with your application that this Merkle key format is used (true
 for applications built w/ Cosmos SDK).
 `,
-	RunE: runProxy,
-	Args: cobra.ExactArgs(1),
-	Example: `light cosmoshub-3 -p http://52.57.29.196:26657 -w http://public-seed-node.cosmoshub.certus.one:26657
+		RunE: func(cmd *cobra.Command, args []string) error {
+			chainID = args[0]
+			logger.Info("Creating client...", "chainID", chainID)
+
+			witnessesAddrs := []string{}
+			if witnessAddrsJoined != "" {
+				witnessesAddrs = strings.Split(witnessAddrsJoined, ",")
+			}
+
+			lightDB, err := dbm.NewGoLevelDB("light-client-db", dir)
+			if err != nil {
+				return fmt.Errorf("can't create a db: %w", err)
+			}
+			// create a prefixed db on the chainID
+			db := dbm.NewPrefixDB(lightDB, []byte(chainID))
+
+			if primaryAddr == "" { // check to see if we can start from an existing state
+				var err error
+				primaryAddr, witnessesAddrs, err = checkForExistingProviders(db)
+				if err != nil {
+					return fmt.Errorf("failed to retrieve primary or witness from db: %w", err)
+				}
+				if primaryAddr == "" {
+					return errors.New("no primary address was provided nor found. Please provide a primary (using -p)." +
+						" Run the command: tendermint light --help for more information")
+				}
+			} else {
+				err := saveProviders(db, primaryAddr, witnessAddrsJoined)
+				if err != nil {
+					logger.Error("Unable to save primary and or witness addresses", "err", err)
+				}
+			}
+
+			trustLevel, err := tmmath.ParseFraction(trustLevelStr)
+			if err != nil {
+				return fmt.Errorf("can't parse trust level: %w", err)
+			}
+
+			options := []light.Option{light.Logger(logger)}
+
+			if sequential {
+				options = append(options, light.SequentialVerification())
+			} else {
+				options = append(options, light.SkippingVerification(trustLevel))
+			}
```

Comment:
Not from you I don't believe. 
```suggestion

			vo := light.SkippingVerification(trustLevel)
			if sequential {
				vo = light.SequentialVerification()
			}
			options = append(options, vo)

```